_summaries/glorot2010understanding_2.md

```diff
@@ -1,19 +1,31 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
-# Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+paper: glorot2010understanding
+author: elampietti
+score: 8/10
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+The core idea of this paper is to challenge the effectiveness of asymmetric activation functions, such as sigmoid, with the traditional, random initialization for deep architecture.
+The paper accomplishes this by providing improved results from their normalized initialization process and by using symmetric activation functions such as hyperbolic tangent and softsign. 
+
+The Shapeset-3 x 2 dataset is used in this paper for the test models to predict 9 possible shape classifications.
+
+The paper makes a point that studying the changes in gradients and activation function values between epochs and network layers is informative towards the effectiveness of a model.
+
+With deeper networks, the variance of the gradient will decrease using the standard initialization as you back-propagate through the network and therefore result in slower learning, therefore they created a normalized initialization to make a more consistent varience in gradient/activation values throughout the network.
```

Comment:
@elampietti Great explanation and context! I would expand on what normalized initialization is, as this was something that they proposed, specifically some of the math behind it. Maybe include the equation for it and discusses how they came up with it.